.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
+# the repo
+* @hazelcast/documentation
```

Comment:
Is there an error signified here? Just wondering if we have the right teams/accounts with admin rights for this repo (I don't)